pkgs/tools/typesetting/tex/texlive/package-setup-hook.sh

```diff
@@ -0,0 +1,12 @@
+{
+  local texFolder
+  local texFolders
+  texFolders=(@texFolders@)
+  for texFolder in "${texFolders[@]}" ; do
+    addToSearchPath TEXMFAUXTREES "$texFolder"
+    if [[ -d "$texFolder/fonts" ]] ; then
+      addToSearchPath OSFONTDIR "$texFolder/fonts"
```

Comment:
OSFONTDIR is interpolated in OPENTYPEFONTS in texmf.cnf as
```
% OpenType outline fonts.
OPENTYPEFONTS = $TEXMFDOTDIR;$TEXMF/fonts/{opentype,truetype}//;$OSFONTDIR//
```
Thus if if OSFONTDIR contains multiple directories, they must be separated by `//:` as otherwise recursion is only enabled for the last one.

Reply:
Actually, I think that all of this can be removed as TEXMFAUXDIRS is already included in TEXMF

Reply:
> Actually, I think that all of this can be removed as TEXMFAUXDIRS is already included in TEXMF

It turns out that my `TEXMFAUXTREES` addition is not doing anything for `luaotfload`, it's just being ignored! There is no other equivalent variable where to fit the extra folder as far as I know. I'll have to consult the resolver code to be sure.

Reply:
...so if I move `$TEXMFAUXTREES` in *front* of `$TEXMFHOME`, everything works out exactly as it should without `OSFONTDIR` (checked with multiple fonts and verified that `TEXMFHOME` is still picked up correctly). As soon as the other test PR is merged, I'll add a test. I won't try to figure out why the position is relevant.